lms/templates/instructor/edx_ace/accountcreationandenrollment/email/body.html

```diff
@@ -31,6 +31,13 @@ <h1>
                 <br />
             </p>
 
+            <p style="color: rgba(0,0,0,.75);">
+		    {% blocktrans with start_link="<a href='{{ course_url }}'>" end_link="</a>" trimmed %}
```

Comment:
@gr4yscale This new interpolation doesn't seem to work. I get (in my devstack, in `cat /edx/src/ace_messages/.20190716-220125.html`):

```
            <p style="color: rgba(0,0,0,.75);">
		    You may access your course at: <a href='{{ course_url }}'>http://localhost:18000/courses/course-v1:edX+DemoX+Demo_Course/</a>.
                <br />
            </p>

            <p style="color: rgba(0,0,0,.75);">
                Sincerely yours, The edX Demonstration Course Team
                <br />
            </p>
```

Reply:
I fixed this.